ceph/rbd/workflows/rbd_mirror.py

```diff
@@ -463,6 +470,9 @@ def config_mirror_multi_pool(
             if (
                 not kw.get("do_not_enable_mirror_on_image")
                 and pool_config.get("mode") == "image"
+            ) or (
+                pool_config.get("mode") == "pool"
```

Comment:
`snapshot` based applicable only for `image` based Is it?

Reply:
Yes, that's why this is a negative scenario test.